pkg/apb/unbind.go

```diff
@@ -70,6 +70,13 @@ func (e *executor) Unbind(
 			e.actionFinishedWithError(err)
 			return
 		}
+		// Delete the binding extracted credential here.
+		err = runtime.Provider.DeleteExtractedCredential(bindingID, clusterConfig.Namespace)
+		if err != nil {
```

Comment:
I think we might want to just log this at an info level if it's a miss error instead of finishing with error here too.